doc/en/deprecations.rst

```diff
@@ -477,3 +471,21 @@ Removed all ``py.test-X*`` entry points. The versioned, suffixed entry points
 were never documented and a leftover from a pre-virtualenv era. These entry
 points also created broken entry points in wheels, so removing them also
 removes a source of confusion for users.
+
+
+``Node.get_marker``
+~~~~~~~~~~~~~~~~~~~
+
+*removed in version 4.1*
+
+As part of a large :ref:`marker-revamp`, :meth:`_pytest.nodes.Node.get_marker` is deprecated. See
+:ref:`the documentation <update marker code>` on tips on how to update your code.
+
+
+``somefunction.markname``
+~~~~~~~~~~~~~~~~~~~~~
+
+* Removed in version 4.1
+
+As part of a large :ref:`marker-revamp` we already deprecated using ``MarkInfo``
+the only correct way to get markers of an element is via ``node.iter_markers([name]``.
```

Comment:
missing closing parenthesis... also I don't think we support a list of names with `iter_markers`, so I suggest using simply: `node.iter_markers(name)` 👍

Reply:
(I mention list of markers because `[name]` might be interpreted as a list of names)

Reply:
good catch, thanks